src/group/libp2p_group_relcast_server.erl

```diff
@@ -213,6 +213,18 @@ handle_cast({request_target, Index, WorkerPid, _WorkerRef}, State=#state{tid=TID
                                              {keys, State#state.group_keys}]}},
     libp2p_group_worker:assign_target(WorkerPid, {Target, ClientSpec}),
     {noreply, NewState};
+handle_cast({clear_target, _Kind, _WorkerPid, Ref}, State=#state{workers = Workers}) ->
```

Comment:
but the target for each worker in the relcast server is always the same.  maybe we should just drop the cast here?

Reply:
Yeah  you are right, relcast doesnt need this but maybe its better to handle the clear target cast as a noop instead, would avoid any future potential confusion if someone is in this flow and wondering why its not handled and doesnt have full context ?

Reply:
yeah, that's what I meant, sorry.  I think a noop clause here with a good comment is the right thing.